src/sxastarter/package.json

```diff
@@ -36,7 +36,7 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@sitecore-feaas/clientside": "^0.3.6",
-    "@sitecore-jss/sitecore-jss-nextjs": "~21.2.0",
+    "@sitecore-jss/sitecore-jss-nextjs": "^21.2.2",
```

Comment:
These JSS dependencies should be using tilde (as of JSS v21 since [minor versions may include breaking changes in framework packages](https://github.com/Sitecore/jss/blob/release/21.0.0/CHANGELOG.md#:~:text=Minor%3A%20may%20include%20breaking%20changes%20in%20framework%20packages%20(e.g.%20framework%20upgrades%2C%20new%20features%2C%20improvements))).
@matkovskyi Looks like this was changed with https://github.com/sitecorelabs/xmcloud-foundation-head-dev/pull/123

Reply:
Resolved with https://github.com/sitecorelabs/xmcloud-foundation-head-dev/pull/126